apps/astarte_housekeeping/lib/astarte_housekeeping/queries.ex

```diff
@@ -196,28 +205,30 @@ defmodule Astarte.Housekeeping.Queries do
 
   defp do_create_realm(
          realm_name,
+         keyspace_name,
          public_key_pem,
          replication_map_str,
          device_limit,
          max_retention
        ) do
     Xandra.Cluster.run(:xandra, [timeout: 60_000], fn conn ->
       with :ok <- validate_realm_name(realm_name),
-           :ok <- create_realm_keyspace(conn, realm_name, replication_map_str),
+           :ok <- create_realm_keyspace(conn, keyspace_name, replication_map_str),
+           {:ok, keyspace_conn} <- build_realm_conn(conn, keyspace_name),
```

Comment:
`realm_conn` here is just used to unpack the realm name and it's not very clear so I'd do this:
- rename `build_realm_conn` to `build_keyspace_conn`
- remove line `218` (not needed)
- pass the `realm_name` explicitly to `insert_realm` (i.e. `insert_realm(conn, realm_name, device_limit)`, it doesn't need the `keyspace_conn` since it acts on the `astarte` keyspace)